src/main/java/com/autotune/utils/KruizeConstants.java

```diff
@@ -674,6 +676,8 @@ public static final class KRUIZE_CONFIG_ENV_NAME {
         public static final String CLOUDWATCH_LOGS_LOG_LEVEL = "logging_cloudwatch_logLevel";
         public static final String LOCAL = "local";
         public static final String LOG_HTTP_REQ_RESP = "logAllHttpReqAndResp";
+        public static final String RECOMMENDATIONS_URL = "recommendationsURL";
```

Comment:
Why do we need `recommendationsURL`?

Reply:
this is required to specifiy where krizue recommendation pod are located . if it is localhost then we need to specify 
service_name.namespace.svc.local.net in manifest which @saad create pr for same

Reply:
PR [1335](https://github.com/kruize/autotune/pull/1335) has the manifest changes for the same